diktat-analysis.yml

```diff
@@ -232,6 +232,26 @@
     maxParametersInOneLine: 2
     # 3 by default.
     # maxCallsInOneLine: 3
+# Checks trailing comma
+- name: TRAILING_COMMA
+  enabled: true
+  configuration:
+    # VALUE_ARGUMENT
+    valueArgument: false
```

Comment:
For our own check I think we wanted to enable it. Does it produce a lot of errors? If yes, open a separate issue for it, otherwise enable and fix here.

Reply:
Yes, there will be many mistakes. Better open a new issue on this and discuss.